package.json

```diff
@@ -14,7 +14,9 @@
     "lint:fix": "npm run lint --silent -- --fix",
     "lint:watch": "watch 'npm run lint --silent' src",
     "postinstall": "npm run build",
-    "test": "ava"
+    "pretest": "npm run lint",
+    "test": "ava",
+    "test:watch": "watch 'npm test --silent' src test"
```

Comment:
Lint is run on `pretest` now, this makes CI a little easier and also `test:watch` feedback is great:

![image](https://cloud.githubusercontent.com/assets/5067638/13732407/6a5557d8-e93f-11e5-82a2-783ac64e6706.png)

Used `watch` here over the built in watch so we can take advantage of the `--silent` flag.  Since the `watch` command runs npm, we get linting in our test watch loop for free.  Also, the `ava` watch wasn't actually working...